INSTALL.rst

```diff
@@ -175,4 +239,33 @@ entry points and then do the following (e.g. for the `hep` rule):
 
 .. code-block:: console
 
-    dojson -l marcxml -i inspirehep/demosite/data/sample.xml do hep
+    (inspirehep)$ dojson -l marcxml -i inspirehep/demosite/data/sample.xml do hep
+
+
+Giving your user admin access
+==============================
+
+If you want access to the admin console, you'll have to execute:
```

Comment:
In the guide it tells you to create that user, I just followed the guide, line 73, has it's own section.

I was unaware of that fixture, is it run by default on install? I can replace the user creation section with this.